packages/react-ui-components/src/enhanceWithClickOutside/index.tsx

```diff
@@ -23,8 +32,20 @@ const enhanceWithClickOutside = <P extends {}>(Component: React.ComponentClass<P
                 (e.target instanceof HTMLElement || e.target instanceof HTMLDocument)
             ) {
                 const domNode = ReactDOM.findDOMNode(this.wrappedInstanceRef.current);
+
+                // if the iframeTarget is set, we ignore the event's target, as there might
+                // be rubbish in there.
+                const target = e.iframeTarget || e.target;
+
+                // Ignore target if it has the data attribute data-ignore_click_outside
+                // with value true. We skip the click outside handling. For further information
+                // take look at https://github.com/neos/neos-ui/issues/2489
```

Comment:
please add the comment in detail here

https://github.com/neos/neos-ui/issues/2489#issuecomment-493538245 (it's very well written, and explains the issue well)

Furthermore, I'd suggest to add the extended description to the commit message as well.